module/VuFind/src/VuFind/Form/Form.php

```diff
@@ -723,6 +725,11 @@ protected function parseConfig($formId, $config, $params)
                 }
             }
 
+            $prefill = $this->sanitizePrefill($prefill);
```

Comment:
Should we do this at the top of the function, so we only sanitize once instead of doing it on every loop iteration?